extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -557,6 +559,11 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
                 }
             }
 
+            if (_self._config.includeResponseErrorData && xhr.status !== 200) {
```

Comment:
>_self._config.includeResponseErrorData && xhr.status !== 200 [](start = 16, length = 60)

Do we want to do this for all !=200 status codes or should we just check >=400 ?

Reply:
+1 we should only do this for status >= 400

As any response from 200 -> 299 is considered to be a success response, a common response is 204 == No Content.
And 300-> 399 are redirects